frontend/js/Pages/Error.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div>
+    <H1>{{ title }}</H1>
+    <div>{{ description }}</div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    status: Number,
+  },
+  computed: {
+    title() {
+      return {
+        503: "503: Service Unavailable",
```

Comment:
Do we translate this?

Reply:
We probably should, but right now this page is unused, because this page wasn't implemented yet.
I think it could be a separate issue.

Reply:
I reintroduced Error page from previous PR and adjusted it to handle more error codes defined in ExceptionHandler.